platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/UnsafeConfigurationResolutionDeprecationIntegrationTest.groovy

```diff
@@ -426,8 +426,7 @@ class UnsafeConfigurationResolutionDeprecationIntegrationTest extends AbstractDe
         """
 
         expect:
-        executer.expectDocumentedDeprecationWarning("Resolution of the configuration :foo was attempted from a context different than the project context. Have a look at the documentation to understand why this is a problem and how it can be resolved. This behavior has been deprecated. This will fail with an error in Gradle 9.0. For more information, please refer to https://docs.gradle.org/current/userguide/viewing_debugging_dependencies.html#sub:resolving-unsafe-configuration-resolution-errors in the Gradle documentation.")
         executer.withArguments("--parallel", "-I", "init-script.gradle")
-        succeeds(":help")
+        fails(":help")
```

Comment:
We should change the title of this test. Right now it says `deprecation warning....`

Also, should we add a `failure.assertHasCause` here?